web/tests/common.test.js

```diff
@@ -23,6 +23,7 @@ run_test("phrase_match", () => {
     assert.ok(common.phrase_match("Tes", "test"));
     assert.ok(common.phrase_match("Tes", "Test"));
     assert.ok(common.phrase_match("tes", "Stream Test"));
+    assert.ok(common.phrase_match("", "test"));
```

Comment:
Would you prefer I move this change to a separate commit? I can do that if so. I added this to make it clear `phrase_match` matches on empty string.